k8s/identityd.yaml

```diff
@@ -21,13 +21,10 @@ spec:
         image: openpolicyagent/opa:0.14.0
         imagePullPolicy: IfNotPresent
         name: opa
-        volumeMounts:
-        - mountPath: /config
-          name: config
-          readOnly: true
       - args:
         - --opa-url=http://127.0.0.1:8181/v1
         - --enable-policies=true
+        - --policies=default
```

Comment:
This approach is definitely better. We have to double check with the `sys.authz` package rego (where we will move the current identity rego policies) as OPA expects them to be available at process startup.

Reply:
I think this problem is same as yahoo/k8s-athenz-identity/pull/23.
The next step, I want to be able to specify the namespace as parameters when deploying, using such as kustomize.

Reply:
This is something we could make configurable through the helm template https://github.com/yahoo/k8s-athenz-identity/pull/14
Like @prabushyam mentioned, we need to see if this approach will be able to work with the `sys.authz` package as well.